### ISSUE TITLE
han-transformer not work

### ISSUE THREAD

**Issue by Continue7777** [brightmart/text_classification#91]
i have tried p1_HierarchicalAttention_model_transformer.py with your test function,but the loss dose not converge。
![image](https://user-images.githubusercontent.com/20719321/48344963-9b114400-e6b1-11e8-9f72-361529bef03c.png)

**Comment by brightmart** [comment]
I think you are run on toy task, and loss is decreasing. if you want coverage, you can use real task,

**Comment by Continue7777** [comment]
I run p1_HierarchicalAttention_model.py ,its work fine .and i just want to try transformer,but i dosen't work,so i try you test function,but it dose not work too?

**Comment by Continue7777** [comment]
thanks

**Comment by brightmart** [comment]
I think transformer work better now. as someone find a bug which was fixed:
https://github.com/brightmart/text_classification/issues/93

**Comment by Continue7777** [comment]
hey,that's me . and i just try your transofrmer code.it's work.    but i campare to textcnn, just as the same conclusion in your github(https://github.com/brightmart/text_classification). transformer isn't better than  textcnn ,may be my dataset is not large enough.  But, i think transformer may be more potential. I will  try pre-train like bert and find whether it's help.

**Comment by brightmart** [comment]
good to hear from you. please keep me update.
